### ISSUE TITLE
Elasticsearch-migration plugin not working on es 2.4.0

### ISSUE THREAD

**Issue by biswajit86** [elastic/elasticsearch-migration#85]
I am trying to run the elasticsearch migration plugin on es-2.4.0 but I ddo  not get any output from the plugin. I do not see any errors on the webpage or in the elastic logs. How can I figure out why it is not working ?

PS:
I do have cors enabled and the http.cors.allow-origin set to "*"

**Comment by MohMaz** [comment]
I have same problem in version 2.3.5.

**Comment by simpleuser99** [comment]
I have same problem in version 2.4.1

**Comment by msergg** [comment]
2.3.2 the same

**Comment by clintongormley** [comment]
Hi @biswajit86 

Are you installing the plugin as described on https://github.com/elastic/elasticsearch-migration/releases/tag/v2.0.1 ?

What browser are you using?  Can you look in the console log of your browser and see if there is anything there?  Upload a screenshot?

**Comment by biswajit86** [comment]
@clintongormley , I work behind a firewall, So i download the plugin offline and install it via the file construct.

I am using chrome browser

I am getting the following error

```
Uncaught SyntaxError: Duplicate data property in object literal not allowed in strict mode : main.js:1546
```

This is line 1546 in main.js 

```
      "index.cache.query.enable" : "index.requests.cache.enable",
```

**Comment by clintongormley** [comment]
thanks @biswajit86 - I'm also using Chrome, weird that mine doesn't complain about the duplicate entry.  This is fixed by e4752b7851c1e0285e9eb910fb4919da6a14ce60

I'll do a new release after fixing some more issues.  thanks for reporting
